libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/maneuver/ManeuverView.java

```diff
@@ -81,32 +80,57 @@
   private String maneuverType = null;
   @ManeuverModifier
   private String maneuverModifier = null;
-  private Pair<String, String> maneuverTypeAndModifier = new Pair<>(null, null);
+  @ColorInt
   private int primaryColor;
+  @ColorInt
   private int secondaryColor;
   private float roundaboutAngle = DEFAULT_ROUNDABOUT_ANGLE;
+  private Pair<String, String> maneuverTypeAndModifier = new Pair<>(null, null);
   private PointF size;
 
+  /**
+   * A custom view that can be used with the Mapbox Directions API.
+   * <p>
+   * By providing a {@link String} maneuver type and maneuver modifier, the
+   * corresponding maneuver icon will be rendered in this view.
+   */
```

Comment:
What about adding Javadoc `@param`s? Same thing happens in the rest of constructors 👇